UPDATING.md

```diff
@@ -2201,7 +2201,7 @@ It is no longer required to set one of the environment variables to avoid
 a GPL dependency. Airflow will now always use text-unidecode if unidecode
 was not installed before.
 
-### new `sync_parallelism` config option in celery section
+### New `sync_parallelism` config option in Celery section
```

Comment:
```suggestion
### New `sync_parallelism` config option in `celery` section
```

Reply:
We are not talking about Celery the tech but `celery` the config section in airflow.cfg